test/e2e/common/configmap.go

```diff
@@ -212,6 +218,7 @@ var _ = ginkgo.Describe("[sig-node] ConfigMap", func() {
 		}
 		framework.ExpectEqual(testConfigMapFound, true, "failed to find ConfigMap in list")
 
+		// hit the endpoint for delete collection
```

Comment:
This hits it but only checks if it returns no error. It does not check the actual functionality - that the config map is gone.

Reply:
Good thought, that make sense.
Should this become a new PR to add in that check?

Reply:
:( yeah, it will need to be a new PR, then we can rebase this on it.

Reply:
has this change been done? it needs to be done and have the non-flakiness verified before we can merge this

Reply:
[Made the PR](https://github.com/kubernetes/kubernetes/pull/89707)